README.md

```diff
@@ -201,3 +201,54 @@ AWS OpsWorks Chef Deployment
   ```ruby
   include_recipe 'datadog::dd-agent'
   ```
+
+Datadog Monitor resource
+==========================
+
+The `datadog_monitor` resource will help you to set up integrations which do not have a recipe yet.
```

Comment:
I don't think we should mention `integrations which do not have a recipe yet` here. What we can do instead is improve the `Instructions` section above to include a step that explains how to enable integrations

Reply:
also, let's mention here that it's for `Agent integrations` (you never know, some users may understand it allows setting up crawler-based integrations :) )